content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -450,6 +450,8 @@ poorly-behaved workloads that may be harming system health.
     queue excess requests, or
   * `time-out`, indicating that the request was still in the queue
     when its queuing time limit expired.
+  * `cancelled`, indicating that the request it is not purge locked
+    and has been ejected from the queue.
```

Comment:
Please revise the sentence. It doesn't flow well...

Reply:
Fixed with the latest commit